sources/vcdMerge/src/SignalSource.cpp

```diff
@@ -97,11 +99,28 @@ void MERGE::SignalSource::Create()
 
 void MERGE::SignalSource::SetFormat(const std::string &rFormat)
 {
-    if (rFormat != "T")
+    if (rFormat == "T")
     {
-        throw EXCEPTION::VcdException(EXCEPTION::Error::INVALID_LOG_FILE_FORMAT,
-                                      "Invalid log file format: " + rFormat);
+        m_pSignalFactory = std::make_unique<PARSER::SignalFactory>();
+        return;
```

Comment:
Why return here? `if` statement below is impossible anyway. Or use `else if`.

Reply:
There was an exception thrown at the very end if any of the formats wasn't detected. But I agree 'else if' will make it more readable.